helpers/emailsender/index.coffee

```diff
@@ -0,0 +1,34 @@
+# TODO move this into a node module
+# TODO make emails instance variables so they can be parameterized
+class EmailSender
```

Comment:
this needs to be pulled into `exports` so it can be actually used when it's required